### PR TITLE
Fix errors when enable_authenticator_manager is not set

### PR DESCRIPTION
**Pull request by l-vo**
Fix errors when enable_authenticator_manager is not set and true is the default value (from Symfony 6)

Related to symfony/recipes#1085

### CODE REVIEW

File: src/Maker/MakeAuthenticator.php [hunk 1]
```diff
@@ -103,7 +104,7 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
         $manipulator = new YamlSourceManipulator($this->fileManager->getFileContents($path));
         $securityData = $manipulator->getData();
 
-        if (!($securityData['security']['enable_authenticator_manager'] ?? false)) {
+        if (interface_exists(GuardAuthenticatorInterface::class) && !($securityData['security']['enable_authenticator_manager'] ?? false)) {
```
Comment (jrushlow) [inline src/Maker/MakeAuthenticator.php:107]:
```suggestion
        // @legacy - Can be removed when Symfony 5.4 support is dropped
        if (interface_exists(GuardAuthenticatorInterface::class) && !($securityData['security']['enable_authenticator_manager'] ?? false)) {
```

File: src/Maker/MakeRegistrationForm.php [hunk 2]
```diff
@@ -124,7 +125,7 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
         $providersData = $securityData['security']['providers'] ?? [];
 
         // Determine if we should use new security features introduced in Symfony 5.2
-        if ($securityData['security']['enable_authenticator_manager'] ?? false) {
+        if (!interface_exists(GuardAuthenticatorInterface::class) || ($securityData['security']['enable_authenticator_manager'] ?? false)) {
```
Comment (jrushlow) [inline src/Maker/MakeRegistrationForm.php:128]:
```suggestion
        // @legacy - Can be removed when Symfony 5.4 support is dropped
        if (!interface_exists(GuardAuthenticatorInterface::class) || ($securityData['security']['enable_authenticator_manager'] ?? false)) {
```

**Review by jrushlow — CHANGES_REQUESTED** [review]
Awesome thanks for this @l-vo If you could add a couple legacy notations, we'll get this merged in!

**Comment by l-vo** [comment]
status: ready to review

**Comment by l-vo** [comment]
@jrushlow legacy comments added 🙂

**Comment by l-vo** [comment]
status: needs review

**Review by jrushlow — APPROVED** [review]
Thank you @l-vo

**Pull request merged**